src/controllers/user.controller.js

```diff
@@ -1,19 +1,24 @@
 import UserService from "../services/user.service.js";
+import customMessage from "../utils/customMessage.js";
 import helper from "../utils/helpers.js";
 import responses from "../utils/responses.js";
 import statusCode from "../utils/statusCode";
 import email from "../utils/email.js";
-import { jwtToken } from "../utils/util.jwt";
+import generateToken, { jwtToken } from "../utils/util.jwt";
 import cloudinary from "../utils/cloudinary";
 import userUpdateValidation from "../validation/userUpdate.validation";
-import customMessage from "../utils/customMessage";
+// import customMessage from "../utils/customMessage";
+// import sendNotificationEmail from "../utils/email.js";
 
+// import UserService from "../services/user.service.js";
```

Comment:
remove unnecessary comments

Reply:
Thanks for the review, I have removed all unnecessary comments.